packages/elements/src/photon-medication-search/photon-medication-search-component.tsx

```diff
@@ -115,7 +115,9 @@ function getFilteredData(
   // Filter the catalog data and treatment options based on the search text
   return catalogData.filter((item) => {
     const itemName =
-      'treatment' in item ? item.treatment.name.toLowerCase() : item.name.toLowerCase();
+      'treatment' in item
+        ? `${item?.name?.toLowerCase() || ''} ${item.treatment.name.toLowerCase()} `
```

Comment:
Is the space here going to screw things up if there is no name?

I think you should just move this into the `every` below

Reply:
the space doesn't have an effect, I've tested terms in both
<img width="242" alt="Screen Shot 2024-09-25 at 2 19 06 PM" src="https://github.com/user-attachments/assets/5ad72d89-18e9-4367-bafb-c1f784112cb8">
<img width="481" alt="Screen Shot 2024-09-25 at 2 19 55 PM" src="https://github.com/user-attachments/assets/9a4fd46e-3f05-4c00-a3dc-773c5daaec05">